docs/UserGuide.md

```diff
@@ -27,38 +31,91 @@ It allows for faster and more effective student management.
+1. Ensure you have **Java 11** or above installed in your computer.
 
-3. Copy the file to the folder you want to use as the home folder for your TutorBuddy application.
+1. Download the latest version of TutorBuddy [here](https://github.com/AY2021S2-CS2103T-T11-1/tp/releases).
 
-4. Double-click the file to start the app. The GUI similar to the below should appear in a few seconds. Note how the app contains some sample data. <br>
-   ![Ui](images/Ui.png)
+1. Move the downloaded TutorBuddy to a folder of your choice. This will be known as the home folder of TutorBuddy.
 
-5. Type the command in the command box and press Enter to execute it. e.g. typing **`list_student`** and pressing Enter will open the list of all students.<br>
+1. Double-click the file to start the application. An application similar to the one below should appear in a few
+   seconds.<br>
 
-   Some example commands you can try:
+![Ui](images/Ui.png)
 
-**Students**
-  * `list_student`: List all students
-  * `find_student James`: Finds and lists all students that have the keyword **James** in the student's name
-  * `add_student n/John Doe p/98765432 e/johnd@example.com a/John street, block 123, #01-01 l/Sec2 g/95421323 r/Mother`: Adds a student John Doe to the Tutor Buddy application
-  * `delete_student 3`: Deletes the 3rd student in the student list
+### Using TutorBuddy
 
-**Tuition Session**
-  * `list_session`: List all tuition sessions
-  * `find_session James`: Finds and lists all tuition sessions that James have
-  * `add_session n/John Doe d/2021-01-01 t/13:00 k/120 s/Biology f/80`: Adds a tuition session for John Doe happening on 14-02-2021
-  * `delete_session n/John Lee i/1`: Deletes the 1st tuition session in John's tuition session list
+This section offers an overview of the layout in TutorBuddy.
+
+There are three main areas in TutorBuddy:
+
+1. the utility area,
+
+1. the main viewing area,
+
+1. the command box and result display box.
+
+![ApplicationOverview](images/ApplicationOverview.png)
+
+1. The utility area consists of 2 tabs: **File**, and **Help**.
+* The **File** tab consists of an exit button. To exit TutorBuddy, click on the exit button. Alternatively, use the `exit` command to exit the application.
+* The **Help** tab consists of a help button. If you require any assistance, click on the help button, copy the link
+  displayed and paste it into any web browser. Alternatively, press <kbd>F1</kbd> to bring up the help window.
+  
+2. The main viewing area consists of 3 tabs: **Home**, **Tuition** and **Calendar**.
+* The **Home** tab contains a reminder section where a user can view their upcoming tuition lessons. There is also
+  a fee section where the user can view their tuition fees for the past 3 months.
+* The **Tuition** tab contains a Student and Session section where a user can view and manage their students and lessons.
+* The **Calendar** tab contains a weekly view of all sessions.
```

Comment:
Should we change 'user' to 'tutor' instead? Might be more "relatable"

Reply:
I think we can just keep 'user' for now, and standardize it later once everyone merge? Because I think some parts of the document might have used 'user' also.

Reply:
ok!